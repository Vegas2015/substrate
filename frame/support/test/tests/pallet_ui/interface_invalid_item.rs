#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::{Interface, PhantomData};

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(PhantomData<T>);

	#[pallet::interface]
	impl<T: Config> Interface for Pallet<T> {}

	#[pallet::call]
	impl<T: Config> Pallet<T> {}
}

fn main() {
}
