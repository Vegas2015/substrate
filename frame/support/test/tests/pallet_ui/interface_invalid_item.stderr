error[E0107]: wrong number of type arguments: expected 1, found 0
  --> $DIR/interface_invalid_item.rs:12:18
   |
12 |     impl<T: Config> Interface for Pallet<T> {}
   |                     ^^^^^^^^^ expected 1 type argument
